cloudsupport/cloudproviderconfiguration_test.go

```
@@ -0,0 +1,72 @@
+			}
+		})
+	}
+}
+
+// AKS path returns ErrCloudDescribeUnavailable (rather than a bare error) when
+// AZURE_SUBSCRIPTION_ID / AZURE_RESOURCE_GROUP are not set. This is the
+// failure mode real air-gapped users hit before any network call is attempted,
+// and it's what lets the kubescape scan loop classify the failure as
+// non-fatal.
+func TestAKSMissingCredsWrapsSentinel(t *testing.T) {
+	os.Unsetenv(KS_OFFLINE_ENV_VAR)
```

golangci-lint (2.11.4)

[error] Error return value of `os.Unsetenv` is not checked

(errcheck)